package.json

```diff
@@ -6,6 +6,10 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "mocha test/index.js",
+    "test:ci": "yarn test && yarn test:ci:graphql-v15.x && yarn test:ci:graphql-v16.0 && yarn test:ci:graphql-v16.x",
+    "test:ci:graphql-v15.x": "yarn upgrade graphql@^15.0.0 && yarn test",
+    "test:ci:graphql-v16.0": "yarn upgrade graphql@=16.0.0 && yarn test",
```

Comment:
I think we can remove this, unless I'm missing something. What's the intention being specifically testing against `v16.0.0`? We should be good with testing against `v16.x`.

Reply:
Only that sometimes a library may not start to work until half way through a range. For example this library works with `14.4.0`, but not with earlier versions of v14. I guess it matters most for the initial version that you're testing against (v15) rather than higher versions, and that test is covered by the naked `yarn test`, since `yarn.lock` is pinned to `15.0.0`, though maybe we should make that more explicit by pinning within `package.json` too.